luigi_pipeline/lib/model/sv_mt_schema.py

```diff
@@ -196,17 +211,37 @@ class SeqrSVGenotypesSchema(SeqrGenotypesSchema):
 
     def _genotype_fields(self):
         is_called = hl.is_defined(self.mt.GT)
+        was_previously_called = hl.is_defined(self.mt.CONC_ST) & ~self.mt.CONC_ST.contains("EMPTY")
+        num_alt = self._num_alt(is_called)
+        prev_num_alt = hl.if_else(
+            was_previously_called,
+            PREVIOUS_GENOTYPE_N_ALT_ALLELES.get(hl.set(self.mt.CONC_ST)),
+            -1,
+        )
+        discordant_genotype = (num_alt != prev_num_alt) & (prev_num_alt > 0)
+        novel_genotype = (num_alt != prev_num_alt) & (prev_num_alt == 0)
         return {
             'sample_id': self.mt.s,
             'gq': self.mt.GQ,
             'cn': self.mt.RD_CN,
-            'num_alt': self._num_alt(is_called)
+            'num_alt': num_alt,
+            'prev_num_alt': prev_num_alt,
+            'new_genotype': hl.or_missing(is_called, discordant_genotype),
+            'new_call': hl.or_missing(is_called, ~was_previously_called | novel_genotype),
```

Comment:
This is a implementation of this truth table:

<img width="479" alt="image" src="https://user-images.githubusercontent.com/5325727/228642289-21d78b70-c279-4690-afc3-0c826bbe314a.png">

Reply:
whether that's correct or not is up in the air.

Reply:
This looks right to me, but what does "Absent Genotype" refer to here? The concordance codes all seem to map to Concordant, Novel, and Discordant

Reply:
Absent Genotype refers to these two:

```
Absent genotype:
0/1	0/0	TN,FN
1/1	0/0	FN
```

in code I didn't think it made sense to split absent and discordant into two categories since the consequent behavior would be the same.

Reply:
Makes sense. Do you mind updating the `PREVIOUS_GENOTYPE_N_ALT_ALLELES`, I like that you have it all grouped by category there but this category is missing